### ISSUE TITLE
Message.toString format discussion

### ISSUE THREAD

**Issue by suntong** [Chatie/wechaty#1676]
Would it be OK that the Message.toString format be changed a bit? 
https://github.com/Chatie/wechaty/blob/ae371981402b12f9a14c44071fbdde0da72c1836/src/user/message.ts#L227-L237

The current format makes it hard for me to ***quickly*** grasp  _who_ has said what at where. 

I know it is much a personal preference, but would it be OK if I change the output format to

    Message#Text[🗣Contact<From Who>@👥Room<Which Room>] The first 70 bytes...

as opposed to,

    Message#Text(👥Room<WhichRoom>🗣Contact<From Who>👤Contact<Always My ID Thus Should be Omitted>)<<img class="qqemoji qqemoji106" text="[捂脸]_web" src="/zh_CN/htmledition/v2/images/spacer.gif" />>

I can do the fix and submit a PR if there is no strong oppose to the change. Thx.

**Comment by huan** [comment]
Yes, it will be ok and I have no opposition to your change proposal.

Please go ahead to submit a PR for you to quickly grasp who has said what at where.

**Comment by suntong** [comment]
Thanks a lot. will do it right away. 
BTW, I had been using '\t' to separate the people@room and message, which makes the messages lining up better, if their name length differ within 7 characters. I'll do this as well, if you don't mind...

**Comment by suntong** [comment]
Oh, @huan, sorry for a noob question -- I've make the changes locally in my `npm root` directory, but the output remains the same. 

Do I need to trigger a rebuild/recompile or something for the change to take effect? If so, how. Thx a lot!

**Comment by suntong** [comment]
> I've made the changes locally in my npm root directory, but the output remains the same.

Please help @huan/@windmemory -- 

I've done the `npm run dist` but the result still remains the same.

Basically, I'm following @windmemory's excellent tutorial article at:

https://github.com/botorange/wechaty-puppet-padpro/wiki/How-to-use-npm-link-to-develop-wechaty-puppet-padpro

and everything works fine, except that the result still remains the same.
Have I changed to the correct place? 

Here is my work log:

```
cd ../wechaty
$ npm run dist
> wechaty@0.23.34 dist /path/to/wx/wechaty
> npm run clean && tsc


> wechaty@0.23.34 clean /path/to/wx/wechaty
> shx rm -fr dist/*

cd ../starter-bot

WECHATY_PUPPET=wechaty-puppet-mock node starter-bot.js 
Message#Text(🗣Contact<mock-name>👤Contact<mock-name>)<mock text>

cd ..
mv starter-bot/ /tmp

cd /tmp/starter-bot/


$ npm root
 /tmp/starter-bot/node_modules
$ grep -A10 'const msgStrList' node_modules/wechaty/src/user/message.ts
    const msgStrList = [
      'Message',
      `#${MessageType[this.type()]}`,
      '[',
        this.from()
          ? 'ð£' + this.from()
          : '',
        this.room()
          ? '@ð¥' + this.room()
          : '',
      ']',


WECHATY_LOG=silly WECHATY_PUPPET=wechaty-puppet-mock node starter-bot.js 
Message#Text(🗣Contact<mock-name>👤Contact<mock-name>)<mock text>

```

How to troubleshoot what went wrong? THX!

**Comment by huan** [comment]
Please follow the issue template and describe the reproduce steps and your expect result in detail, or I can not understand what you want to do.

Please also open a new issue with your new problem, thanks!
